src/application-base/ApplicationBase.jsx

```diff
@@ -9,20 +9,24 @@ import ApplicationErrorBoundary from '../application-error-boundary';
 import ApplicationLoadingOverlay, { ApplicationLoadingOverlayProvider } from '../application-loading-overlay';
 import { NavigationPromptCheckpoint } from '../navigation-prompt';
 import { ApplicationIntlProvider } from '../application-intl';
+import getBrowserLocale from '../browser-locale';
 
 import styles from './ApplicationBase.module.scss';
 
 const cx = classNames.bind(styles);
 
+const browserLocale = getBrowserLocale();
+
 const propTypes = {
   /**
    * The components to render within ApplicationBase.
    */
   children: PropTypes.node.isRequired,
   /**
    * The locale name to be used to load translated messages.
+   * If not provided the preferred language from the browser will be attempted to be used.
```

Comment:
Nitpicking the props table description here:
```suggestion
   * If the `locale` prop is not provided, the preferred language from the browser will be used.
```